packages/kbn-test/src/es/es_test_cluster.js

```diff
@@ -78,7 +79,9 @@ export function createEsTestCluster(options = {}) {
         esArgs: [
           `cluster.name=${clusterName}`,
           `http.port=${port}`,
+          `node.name=${nodeName}`,
           `discovery.zen.ping.unicast.hosts=localhost:${port}`,
+          `cluster.initial_master_nodes=${nodeName}`,
```

Comment:
This could reasonably be `127.0.0.1` and then you don't need to specify the `node.name`.